main.go

```diff
@@ -15,64 +15,97 @@ package main
 
 import (
 	"context"
-	"flag"
+	"fmt"
+	"os"
 
 	"github.com/mendersoftware/go-lib-micro/config"
 	"github.com/mendersoftware/go-lib-micro/log"
+	"github.com/urfave/cli"
 
 	"github.com/mendersoftware/useradm/store/mongo"
 )
 
 func main() {
-	var configPath string
-	var printVersion bool
-	var devSetup bool
 	var debug bool
+	var configPath string
 
-	flag.StringVar(&configPath, "config",
-		"",
-		"Configuration file path. Supports JSON, TOML, YAML and HCL formatted configs.")
-	flag.BoolVar(&printVersion, "version",
-		false, "Show version")
-	flag.BoolVar(&devSetup, "dev",
-		false, "Use development setup")
-	flag.BoolVar(&debug, "debug",
-		false, "Enable debug logging")
-
-	flag.Parse()
+	app := cli.NewApp()
```

Comment:
It's good to set `app.Usage` too (otherwise description in help is the default 'a new cli application')